[2022-02-13 00:00:06,379] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C scheduled__2022-02-12T00:00:00+00:00 [queued]>
[2022-02-13 00:00:06,439] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C scheduled__2022-02-12T00:00:00+00:00 [queued]>
[2022-02-13 00:00:06,439] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-13 00:00:06,439] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-13 00:00:06,440] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-13 00:00:06,521] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): training_model_C> on 2022-02-12 00:00:00+00:00
[2022-02-13 00:00:06,569] {standard_task_runner.py:52} INFO - Started process 1348 to run task
[2022-02-13 00:00:06,579] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_C', 'scheduled__2022-02-12T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpqp5054m4', '--error-file', '/tmp/tmplclq2i87']
[2022-02-13 00:00:06,579] {standard_task_runner.py:77} INFO - Job 18: Subtask training_model_C
[2022-02-13 00:00:07,000] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.training_model_C scheduled__2022-02-12T00:00:00+00:00 [running]> on host f9b316006a9e
[2022-02-13 00:00:07,930] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2022-02-12T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-12T00:00:00+00:00
[2022-02-13 00:00:07,945] {python.py:175} INFO - Done. Returned value was: 2
[2022-02-13 00:00:08,210] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_C, execution_date=20220212T000000, start_date=20220213T000006, end_date=20220213T000008
[2022-02-13 00:00:08,401] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-13 00:00:08,782] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
